Creates tournament attributes dictionary for players

meatballs opened the pull request:
Fixes #286

drvinceknight commented on axelrod/tests/unit/test_tournament.py:
```diff
@@ -49,7 +49,8 @@ def test_init(self):
         self.assertEqual(len(tournament.players), len(self.players))
-        self.assertEqual(tournament.players[0].tournament_length, 200)
+        self.assertEqual(
+            tournament.players[0].tournament_attributes['length'], 200)
```
Could we include a test here for the game attribute also?

And perhaps include tests for non default values?

meatballs commented:
I've added a test for the game value in the tournament attributes dictionary and I've also added a non-default 'turns' value.

I don't think non-default game values belong here. If we want them, they should be in test_game.py and that's not really part of this ticket.

drvinceknight commented:
> I've added a test for the game value in the tournament attributes dictionary and I've also added a non-default 'turns' value.
> 
> I don't think non-default game values belong here. If we want them, they should be in test_game.py and that's not really part of this ticket.

Yeah, sounds good to me. 

I'm happy with this :+1:
